doc/api/deprecations.md

```diff
@@ -54,10 +57,11 @@ changes:
     description: Runtime deprecation.
 -->
 
-Type: Runtime
+Type: End-of-Life
 
-The `OutgoingMessage.prototype.flush()` method is deprecated. Use
-`OutgoingMessage.prototype.flushHeaders()` instead.
+The `OutgoingMessage.prototype.flush()` method was deprecated in Node.js
+1.6.0 and has since been removed. Use `OutgoingMessage.prototype.flushHeaders()`
```

Comment:
Nit: There was no Node.js 1.6.0. It was io.js back then. Probably best to leave versions out of it entirely as it's in the YAML matter and that gets rendered in our docs?